src/index.ts

```diff
@@ -32,6 +32,7 @@ export { BlankImage } from './components/Image'
 export { Heading } from './components/Heading'
 export { HeadlineArea } from './components/HeadlineArea'
 export { Select } from './components/Select'
+export { Accordion, AccordionContent, AccordionTrigger } from './components/Accordion'
```

Comment:
[MUST] Can't build due to error. `TS2307: Cannot find module './components/Accordion'.`

Reply:
I'm sorry for my mistakes.
Fixed this commit. -> https://github.com/kufu/smarthr-ui/pull/408/commits/1d0dd7516b2c234721876c0173e900dfb084b1d5